roles/openshift_aws/defaults/main.yml

```diff
@@ -144,10 +151,7 @@ openshift_aws_master_group_config:
     min_size: 3
     max_size: 3
     desired_size: 3
-    tags:
-      host-type: master
-      sub-host-type: default
-      runtime: docker
+    tags: "{{ openshift_aws_master_group_tags | combine(openshift_aws_master_group_default_tags) }}"
```

Comment:
reverse order of combine.  I believe the second overrides the first.  We want the defaults to be overridden by the user-supplied version.

Reply:
I assumed the opposite since some of these tags make the rest of the code work.

Reply:
User is always right.  They should be able to override anything that is a variable.  We should put a note about the default tags to indicate as such.

Perhaps we should also rename the defaults to 'mandatory_tags' so it's clear that those tags need to be present.